src/main/java/MainPage.java

```diff
@@ -0,0 +1,123 @@
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainPage extends BasePage {
+
+    @FindBy(xpath = ".//div[@id = 'auth-user-links']")
+    private WebElement loginLink;
+
+    @FindBy(id = "header-user-link")
```

Comment:
You should use only one pattern for elelment locators. So please, use only **XPath** for element locators

Reply:
Done